### DISCUSSION TITLE
Adding another stylesheet (IBM carbon) breaks daisyUI theme.

### DISCUSSION THREAD

**Discussion by thenbe** [saadeghi/daisyui#354] [Q&A] [answered]
I'm relatively new to web development in general but I've tried searching for an answer couldn't find one. Basically I'm using tailwind + daisyUI in a sveltekit project and it's going great. However after adding a carbon component and the carbon stylesheet, my daisyUI theme isn't working as expected. Things such as colors and borders are breaking.

Where should I look for a solution to this? Are there any concepts that I should read about to solve this issue? Is it even solvable?

If someone can point me in the right direction I'd be grateful. Thanks.

**Comment by saadeghi** [comment, marked as answer]
The right direction would be picking only one component library for your project. not two.  
However, if you really have to use them both in one project, you can set a [`prefix`](https://tailwindcss.com/docs/configuration#prefix) for your tailwind setup so all Tailwind classes (and daisyUI classes) will be available with a custom prefix (ex: `tw-bg-red-400` instead of `bg-red-400`. or `tw-btn` instead of `btn`). This way you can make sure there are no conflicts with selector names.

**Reply by thenbe** [reply]
Thanks for the quick reply and the awesome library! I especially like the theming aspect.
